setup-di-compile/action.yml

```diff
@@ -0,0 +1,60 @@
+name: "Magento compilation (setup:di:compile)"
```

Comment:
We need a test of this action here: https://github.com/mage-os/github-actions/blob/main/.github/workflows/_internal-setup-di-compile.yaml

Reply:
@damienwebdev @Tjitse-E the target of this link doesn't exist anymore, what should we do here?